docs/docs/installation/github.md

```diff
@@ -28,13 +28,23 @@ jobs:
 ```
 
 
-if you want to pin your action to a specific release (v2.0 for example) for stability reasons, use:
+if you want to pin your action to a specific release (v0.23 for example) for stability reasons, use:
 ```yaml
 ...
     steps:
       - name: PR Agent action step
         id: pragent
-        uses: Codium-ai/pr-agent@v2.0
+        uses: docker://codiumai/pr-agent:0.23-github_action
+...
+```
+
+For enhanced security, you can also specify the Docker image by its digest:
+```yaml
+...
+    steps:
+      - name: PR Agent action step
+        id: pragent
+        uses: docker://codiumai/pr-agent@sha256:14165e525678ace7d9b51cda8652c2d74abb4e1d76b57c4a6ccaeba84663cc64
```

Comment:
add a link to digest, to explain how you found the SHA. It is not trivial

[digest](https://hub.docker.com/layers/codiumai/pr-agent/0.23-github_action/images/sha256-14165e525678ace7d9b51cda8652c2d74abb4e1d76b57c4a6ccaeba84663cc64?context=repo)